src/test/unit/test.js

```diff
@@ -59,6 +60,23 @@ test("Instantiate", function () {
               "Type of first WYMeditor instance, using jQuery.wymeditors(0)");
 });
 
+asyncTest("Custom dialog options don't change future defaults", function () {
```

Comment:
This tests the bug with `DEFAULT_DIALOG_OPTIONS` in-place modification. It failed until I added the `{}`.